retos/Reto-1/rivera1479/index.html

```diff
@@ -0,0 +1,54 @@
+<script src="https://unpkg.com/vue@3/dist/vue.global.js"></script>
+
+<div id="app">
+  <h1>{{ title }}</h1>
+  <input v-model="inputNumber" placeholder="Ingrese el numero" />
+  <br />
+  <button @click="functionFibonacci">Calcular</button> <br />
+  <span>{{ result }}</span>
+</div>
+
+<script>
+  const { createApp, ref } = Vue;
+
+  createApp({
+    setup() {
+      const title = ref("Funcion de Fibonacci");
+      const inputNumber = ref(0);
+      const result = ref([]);
+      const functionFibonacci = function () {
+        result.value = [];
+        if (isNaN(parseInt(inputNumber.value))) {
+          alert("No es un numero " + inputNumber.value);
+          inputNumber.value = 0;
+          return;
+        }
+        if (inputNumber.value === 1 || inputNumber.value === 0) {
+          result.value = [1];
+          inputNumber.value = 0;
+          return;
+        } else {
```

Comment:
Having an empty return in each if case, do you really need the else clause? Take a look if you can just have the else block code as a "default" case without needing the else